Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1030,21 +1030,12 @@ class BrowserViewController: UIViewController, SearchBarLocationProvider, Themea
         addChild(viewController)
         contentContainer.add(content: viewController)
         viewController.didMove(toParent: self)
-        manageStatusBarEmbedded()
+        statusBarOverlay.resetState()
 
         UIAccessibility.post(notification: UIAccessibility.Notification.screenChanged, argument: nil)
         return true
     }
 
-    /// Status bar overlay needs to be at the back for some content type that need extended content
-    private func manageStatusBarEmbedded() {
```

Comment:
This was the original solution to fix the status bar overlay with the homepage bottom URL bar